src/components/Facility/PatientNoteCard.tsx

```diff
@@ -45,25 +44,15 @@ const PatientNoteCard = ({
   const [showEditHistory, setShowEditHistory] = useState(false);
   const [editHistory, setEditHistory] = useState<PatientNotesEditModel[]>([]);
   const authUser = useAuthUser();
-  const queryClient = useQueryClient();
 
-  const { data, refetch } = useQuery({
-    queryKey: [patientId, note.id],
-    queryFn: query(routes.getPatientNoteEditHistory, {
+  const fetchEditHistory = async () => {
+    const { res, data } = await request(routes.getPatientNoteEditHistory, {
       pathParams: { patientId, noteId: note.id },
-    }),
-  });
-
-  const { mutate: updateNote } = useMutation({
-    mutationFn: mutate(routes.updatePatientNote, {
-      pathParams: { patientId, noteId: note.id },
-    }),
-    onSuccess: () => {
-      Success({ msg: "Note updated successfully" });
-      queryClient.invalidateQueries({ queryKey: ["notes", patientId] });
-      setIsEditing(false);
-    },
-  });
+    });
+    if (res?.status === 200) {
+      setEditHistory(data?.results ?? []);
+    }
+  };
```

Comment:
when resolving conflicts, make sure resolve by understanding what's changed in the base branch instead of just reverting all those changes that were already made in the base branch